Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControlTypes.cs

```diff
@@ -35,7 +35,19 @@ public enum HandleType
     {
         None = 0,
         Rotation,
-        Scale
+        Scale,
+        Translation
+    }
+
+    /// <summary>
+    /// Flags which determine which handle type is enabled.
+    /// </summary>
+    [System.Flags]
+    public enum HandleFlags
+    {
+        Rotation = 1 << 0,
```

Comment:
@thalbern Flags require the 1,2,4,8 values. If we re-defined the values of HandleTypes we could, but I thought this would be better. Separate flags enums are already used elsewhere in the codebase.

Reply:
Flags removed!